storage/mysql/admin_storage_test.go

```diff
@@ -151,7 +150,7 @@ func TestAdminTX_StorageSettingsNotSupported(t *testing.T) {
 	s := NewAdminStorage(DB)
 	ctx := context.Background()
 
-	settings, err := anypb.New(&keyspb.PEMKeyFile{})
+	settings, err := anypb.New(&trillian.Tree{})
```

Comment:
this is an interesting substitution. have not looked at the detailed implementations, though.

Reply:
All that this wants is a valid non-nil "any" protobuf, the content is not actually looked at. The name of the field is `storage_settings`, so putting in a `Tree` is no less "reasonable" than putting in a `PEMKeyFile` (they're both equally nonsensical!). 😃

Reply:
Yeah, I was wondering. The comment in tree_validation_test.go made me 71% sure this could be anything. Also the `anypb.New()`.